tests/test_code_report.py

```diff
@@ -405,3 +405,29 @@ def test_code_report_extended_arg_search_embedded(tmp_path: pathlib.Path, stdout
 
     env.run()
     stdout_checker.assert_absent_calls_with_param("${CODE_REPORT_FILE}")
+
+
+@pytest.mark.parametrize('analyzer, extra_args, tested_content', [
```

Comment:
Now you don't need to parametrize the test, because you only have one set of parameter values.

Reply:
Thank you, removed